helix-rest/src/main/java/org/apache/helix/rest/metadatastore/accessor/ZkRoutingDataWriter.java

```diff
@@ -77,7 +88,23 @@ public synchronized boolean addMetadataStoreRealm(String realm) {
       return createZkRealm(realm);
     }
 
-    // TODO: Forward the request to leader
+    String leaderHostName = _leaderElection.getCurrentLeaderInfo().getId();
+    String url =
+        leaderHostName + "/admin/v2/namespaces/" + _namespace + "/metadata-store-realms/" + realm;
+    HttpPut putRequest = new HttpPut(url);
+    try {
+      HttpResponse response = _forwardHttpClient.execute(putRequest);
+      if (response.getStatusLine().getStatusCode() != 201) {
```

Comment:
I would use created code rather than the hard coded 201: easier to read and maintain

Reply:
+1. Don't we already have this as part of Helix REST? See `javax.ws.rs.core.Response;`?

Reply:
Sure.